.github/workflows/php-test-plugins.yml

```diff
@@ -68,20 +73,51 @@ jobs:
       - name: Build assets
         run: npm run build
       - name: Install WordPress
-        run: npm run wp-env start
+        run: |
+          if [ "${{ matrix.coverage }}" == "true" ]; then
+            npm run wp-env start -- --xdebug=develop,coverage
+          else
+            npm run wp-env start
+          fi
       - name: Composer Install
         run: npm run wp-env run tests-cli -- --env-cwd="wp-content/plugins/$(basename $(pwd))" composer install --no-interaction --no-progress
+      - name: Update Composer Dependencies
+        run: composer update --with-all-dependencies --no-interaction --no-progress
+      - name: Install PHPUnit
+        run: |
+          if [ "${{ matrix.php }}" == "7.2" ]; then
+            composer require --dev phpunit/phpunit:${{ matrix.phpunit }} --with-all-dependencies
```

Comment:
This is needed because `composer.lock` may have dependencies which require PHP 7.3?